spec/controllers/api/v1/widget/contacts_controller_spec.rb

```diff
@@ -87,4 +87,25 @@
       end
     end
   end
+
+  describe 'POST /api/v1/widget/destroy_custom_attributes' do
+    let(:params) { { website_token: web_widget.website_token, identifier: 'test', custom_attributes: ['test'] } }
+
+    context 'with invalid website token' do
+      it 'returns unauthorized' do
+        patch '/api/v1/widget/destroy_custom_attributes', params: { website_token: '' }
```

Comment:
The request says it is post, but what is being tested here is a patch

Reply:
Fixed